app/email.go

```diff
@@ -435,13 +435,17 @@ func (a *App) SendGuestInviteEmails(team *model.Team, channels []*model.Channel,
 			data := model.MapToJson(props)
 
 			if err := a.Srv.Store.Token().Save(token); err != nil {
-				mlog.Error(fmt.Sprintf("Failed to send invite email successfully err=%v", err))
+				mlog.Error("Failed to send invite email successfully ", mlog.Err(err))
 				continue
 			}
 			bodyPage.Props["Link"] = fmt.Sprintf("%s/signup_user_complete/?d=%s&t=%s", siteURL, url.QueryEscape(data), url.QueryEscape(token.Token))
 
 			if !*a.Config().EmailSettings.SendEmailNotifications {
-				mlog.Info(fmt.Sprintf("sending invitation to %v %v", invite, bodyPage.Props["Link"]))
+				mlog.Info("sending invitation ", mlog.String("to", invite), mlog.String("link", bodyPage.Props["Link"].(string)))
+			}
+
+			if err := a.SendMail(invite, subject, bodyPage.Render()); err != nil {
```

Comment:
Why this new lines are here?

Reply:
@jespino I will fix that, thanks.

Reply:
@jespino Can you be a bit more explicit? there is no newline between `line 447 and 448`

Reply:
The block:
```
if err := a.SendMail(invite, subject, bodyPage.Render()); err != nil {
	mlog.Error("Failed to send invite email successfully ", mlog.Err(err))
}
```
appears like a new set of lines in the diff of github

Reply:
<img width="723" alt="Screenshot 2019-10-04 at 9 11 25 PM" src="https://user-images.githubusercontent.com/12391886/66236915-9dc2c080-e6eb-11e9-91ca-799d69686198.png">

This is how it looks like in my code editor, what do you think about this?

Reply:
@jespino it looks okay to me in the unified view 
<img width="1028" alt="Screen Shot 2019-10-07 at 10 59 48 AM" src="https://user-images.githubusercontent.com/44858354/66336280-a4228980-e8f1-11e9-8326-f0a22763d475.png">

Reply:
If you go here: https://github.com/mattermost/mattermost-server/blob/master/app/email.go#L443 the same point in master you will see that the lines:

```go
			if !*a.Config().EmailSettings.SendEmailNotifications {
				mlog.Info("sending invitation ", mlog.String("to", invite), mlog.String("link", bodyPage.Props["Link"].(string)))
			}
```

exists, but the lines after:

```go
if err := a.SendMail(invite, subject, bodyPage.Render()); err != nil {
	mlog.Error("Failed to send invite email successfully ", mlog.Err(err))
}
```

doesn't exist.

Reply:
@Lumexralph Any news about this change?

Reply:
@jespino This was the situation I found myself concerning this and you're right concerning the change you noticed.

When I finished working on the task, I raised a PR then realized it had a merge conflict, I tried to update my work and resolve the conflict but then a new function was added to my work, I didn't know what to do and since it was a new function, I decided to keep the new change.

What do you suggest I do?

Reply:
You should remove that new, not needed lines, probably that lines appears there as result of not correct merge where the new version removed it and the old one still have it, but with changes.

Thats it, remove that extra lines and it should be correct.

Reply:
@jespino I will work on that, thanks.

Reply:
@jespino Please, kindly check to ensure the feedback has been implemented.

Reply:
Yes, perfect, now looks good to me :)